wordpress-mobile/WordPress-iOS
# Reader Comments: Remove newCommentThread flag

Fixes n/a

This removes the `newCommentThread` flag. Some files are also wiped from existence, since we will no longer use them: `ReaderCommentCell` and `ReaderPostHeaderView`. 

Note: I'll create another PR to remove the `commentThreadModerationMenu` flag.

## To test

- Go to Reader Comments.
- Verify that the new Comment view is displayed, but the body is still rendered with the old mechanism.

## Regression Notes
1. Potential unintended areas of impact
Should be none.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manually testing to ensure no side effects after the flag is removed.

3. What automated tests I added (or what prevented me from doing so)
n/a.

PR submission checklist:
